roles/prometheus/server/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: systemctl daemon-reload
+  systemd:
```

Comment:
With Ansible Core 2.15, I think it is better to use absolute path in Ansible modules' names:
```ansible.builtin.systemd```